flow-base-api/src/main/java/com/aevi/sdk/flow/BaseApiClient.java

```diff
@@ -136,8 +137,8 @@ public void run() throws Exception {
                 });
     }
 
-    protected ObservableMessengerClient getMessengerClient(ComponentName componentName) {
-        return new ObservableMessengerClient(context, componentName);
+    protected ChannelClient getMessengerClient(ComponentName componentName) {
+        return Channels.webSocket(context, componentName);
```

Comment:
bleep blarp - do we want to make this configurable from the outside? What if we discover some serious problem with websockets on some device once it's all out there?